init.sh

```diff
@@ -1,44 +1,50 @@
-#!/usr/bin/env ruby
+#!/usr/bin/env node
```

Comment:
This file no longer has to be invoked via shell. Since it's all JS, you can just `require` and run it from `cli.js`